interchange/DeviceResources.capnp

```diff
@@ -254,6 +261,7 @@ struct Device {
       conventional @5 : Void;
       pseudoCells  @6 : List(PseudoCell);
     }
+    subTile      @7 : TileTypeSubTileIdx;
```

Comment:
> What if there is no sub-tile index required for a PIP?

Suggestion - if subtiles are present for a tile then all PIPs must have a valid `subTile`. Otherwise `subTile` is ignored - this will be best for backward compatibility imo.

Reply:
Updated comments to this effect